### PR TITLE
Add missing title for Using Plugins page

### PR DESCRIPTION
**Pull request by marvinchin**
**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [x] Documentation update

**What is the rationale for this request?**
Page title is currently missing.

screenshot 2019-02-20 at 9 39 16 pm

**What changes did you make? (Give an overview)**
Add page title.

### CODE REVIEW

File: docs/userGuide/usingPlugins.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
+<variable name="title">Using Plugins</variable>
+<variable name="filename">usingPlugins</variable>
```
Comment (Chng-Zhi-Xuan) [inline docs/userGuide/usingPlugins.md]: If the `{{ filename }}` variable isn't used, we should not declare it. 

See `tipsAndTricks.md`

Comment (marvinchin) [reply docs/userGuide/usingPlugins.md]: Fixed!

File: docs/userGuide/usingPlugins.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
+<variable name="title">Using Plugins</variable>
 <frontmatter>
```
Comment (yamgent) [inline docs/userGuide/usingPlugins.md]: Put a blank line between `<variable>` and `<frontmatter>`.

Comment (marvinchin) [reply docs/userGuide/usingPlugins.md]: Fixed!

There is no blank line between `<variable>` and `<frontmatter>` in `tipsAndTricks.md`. Should we fix that in this PR, or make a separate one for that?

Comment (yamgent) [reply docs/userGuide/usingPlugins.md]: > Should we fix that in this PR, or make a separate one for that?

I don't mind fixing it in this PR.

Comment (Chng-Zhi-Xuan) [reply docs/userGuide/usingPlugins.md]: Then we would need to add a blank line to `tipsAndTricks.md` as well :P

**Comment by marvinchin** [comment]
@yamgent @Chng-Zhi-Xuan issues should be fixed! 🙂

**Pull request merged**